GiniVision/Classes/CameraViewController.swift

```diff
@@ -241,7 +241,7 @@ public typealias CameraErrorBlock = (_ error: CameraError) -> ()
             if GiniConfiguration.DEBUG {
                 // Retrieve image from default image view to make sure image was set and therefor the correct states were checked before.
                 if let image = self.defaultImageView?.image,
-                   let imageData = UIImageJPEGRepresentation(image, 1) {
+                   let imageData = UIImageJPEGRepresentation(image, 0.2) {
```

Comment:
The camera framework doesn't compress the image, yes. However, if you look several lines down, where the real camera image is taken, it immediately passes through `ImageMetainfomationManager` which in turn compresses it. So leaving the `CameraViewController`, the image is always compressed. I changed it to 0.2 because it was a bit strange to see 1.5 MB images while debugging on the simulator.